### DISCUSSION TITLE
Is it possible to check prooved attributes?

### DISCUSSION THREAD

**Discussion by oadamovskyi** [openwallet-foundation/credo-ts#1251] [Q&A] [answered]
I was wondering whether exists already implemented in AFJ feature to check what values of attributes Prover passed.

Let`s assume Issure requested a proofe with attribute "age", then it goes to Holder and he provides some Credential record with proper attribute, for example Credential saying our imaginary Holder is 21 y.o. Is it possible, as an Issuer, check the value in credential (21 y.o) to sell a beer if it is 21 or higher?

Previously I thought it will be stored in ProofRecord received by agent.proofs.getById() when the state is "done", but I was wrong and the field "metadata", where I was seeking for value, in message was empty

**Comment by TimoGlastra** [comment, marked as answer]
Hi @oadamovskyi, you can access the specific data that was exchanged using `agent.proofs.getFormatData` and pass the proofExchangeId. 

This will return an object with the different messages and should include the presentation data

**Reply by oadamovskyi** [reply]
Perfect! It gave me what I desired

Thanks for fast reply
